terms/folding_scheme.md

```diff
@@ -0,0 +1,9 @@
+# Folding Scheme
+
```

Comment:
I think you need to write about what this part is

Reply:
Add the definition of a folding scheme

Reply:
I will add in the next commit.